site/templates/collection/carousel/item/image.jet

```diff
@@ -1,5 +1,14 @@
 {{block carouselItemImage()}}
   {{if isset(.Images["Carousel"])}}
-    <s72-image src="{{.Images.Carousel}}" alt="{{.Title}}" class="carousel-item-image"></s72-image>
+    {* If carousel focus is set for film (film infos metadata) or tv season (tv seasons metadata) or custom fields for Pages / Bundles then it will set a class accordingly to position the carousel image *}
+      {{carouselFocus = ""}}
+      {{if isset(.InnerItem["CarouselFocus"]) && .InnerItem["CarouselFocus"] != "" }}
```

Comment:
See my [comment on the kibble PR](https://github.com/shift72/kibble/pull/138#pullrequestreview-1941776471).  You could change this to something like:
```
{{carouselFocus := .InnerItem.GetCarouselFocus()}}

<s72-image
  src="{{.Images.Carousel}}"
  alt="{{.Title}}"
  class="carousel-item-image {{carouselFocus}}"
/>
```
You don't need that "carousel-focus-" class prefix... I think you can probably use the nesting selector to drop the carouselFocus value straight onto the [existing `img` css](https://github.com/shift72/core-template/blob/d9f20ff0033f54ee37b1f87e1a54829a0f6332ca/site/styles/_carousel.scss#L42) e.g:
```
img {
  height: 100%;
  object-fit: cover;
  object-position: var(--carousel-image-object-position);
  width: 100%;
  z-index: -1;
  @include media-breakpoint-up(sm) {
    object-position: var(--carousel-image-object-position-sm);
  }
  @include media-breakpoint-up(lg) {
    object-position: var(--carousel-image-object-position-lg);
  }

  &.left {
    object-position: left;
  }

  &.right {
    object-position: right;
  }

  &.center {
    object-position: center;
  }
}
```
Or something like that anyway...

Reply:
im getting a linting error trying to use &.left etc off the img selector

Reply:
It's possible that I am misremembering the syntax for that... what's the error?

Reply:
https://stylelint.io/user-guide/rules/selector-no-qualifying-type/

Reply:
Ohhh I think that's because it's an element selector `img` rather than a class selector.  I'm not sure if anything depends on that being `img`, but you might get away with just changing `img` to `.carousel-item-image`.

Reply:
Just adding my 2c. I prefer to get a bit more specific around the name of classes to make it clear. `.left` could mean a few things. Like is the position of this image to the left.

I'm guessing the style rule would end up looking like: `.carousel-item-link > s72-image img.left` which looks ok, the way we would use the class would be:

```
<img src="..." alt="..." class="carousel-item-image left" />
```

Maybe a generic class `.focus-left` that wasn't nested would make more sense?

Caveat to this feedback is I'm not very familiar with the pattern core-template has for css. Best to go with the standard.

Reply:
ive changed it based on @PeculiarGoat feedback. focus-left etc would make it more obvious what its doing, it would mean i would have to add some basic logic to build the classname string. im on the fence, @PeculiarGoat thoughts?

Reply:
ended up using focus-left style after discussion with jack. the logic protects bs coming through and makes it findable in the template code to where this class is set.

Reply:
@jacobtoye @PeculiarGoat  re-review pls when it suits